spec/02-integration/05-proxy/21-reports_spec.lua

```diff
@@ -223,16 +367,73 @@ for _, strategy in helpers.each_strategy() do
                                        ":" .. helpers.get_proxy_port(true) .. "/up-ws")
 
       reports_send_ping()
-      local _, reports_data = assert(reports_server:stop())
 
+      local _, reports_data = assert(reports_server:stop())
       assert.same(1, #reports_data)
       assert.match("requests=1", reports_data[1])
       assert.match("http_reqs=0", reports_data[1])
       assert.match("https_reqs=0", reports_data[1])
+      assert.match("h2c_reqs=0", reports_data[1])
+      assert.match("h2_reqs=0", reports_data[1])
       assert.match("grpc_reqs=0", reports_data[1])
       assert.match("grpcs_reqs=0", reports_data[1])
       assert.match("ws_reqs=0", reports_data[1])
       assert.match("wss_reqs=1", reports_data[1])
     end)
+
+    it("#stream reports tcp streams", function()
+      local tcp = require "socket".tcp()
+      assert(tcp:connect(helpers.get_proxy_ip(false), 19000))
+
+      -- TODO: we need to get rid of the next line!
```

Comment:
Can you clarify this TODO? I don't understand if "getting rid of the next line" mean "the next source code line" (`assert(tcp:send(MESSAGE))`), or "removing part of the body sent via TCP". I would like to know *why* why we would need to do any of those two (or a third one that I haven't imagined).

Reply:
@kikito currently our `tcp` streams won't work, if you are not sending any data first. Don't remember the right number of bytes you need to send to get it to work.